extensions/parquet/table/src/test/java/io/deephaven/parquet/table/metadata/TestRowGroupInfo.java

```diff
@@ -41,13 +41,12 @@ public void testEqualities() {
                 RowGroupInfo.splitEvenly(1),
                 RowGroupInfo.byGroup("a", "b"),
                 RowGroupInfo.byGroup(99, "a", "b"));
-        for (final RowGroupInfo config : distinctConfigs) {
-            assertEquals(String.format("%s == %s", config, config), config, config);
-        }
         final int L = distinctConfigs.size();
         for (int i = 0; i < L - 1; ++i) {
+            final RowGroupInfo config = distinctConfigs.get(i);
+            assertEquals(String.format("%s == %s", config, config), config, config);
```

Comment:
The outer for loop only goes to L - 1, so we are missing assertEquals for the last element.